.travis.yml

```diff
@@ -67,4 +67,4 @@ install:
   - yarn install
 
 script:
-  - yarn ember try:one $EMBER_TRY_SCENARIO --skip-cleanup
+  - node_modules/.bin/ember try:one $EMBER_TRY_SCENARIO --skip-cleanup
```

Comment:
@cibernox why?

Reply:
It's the autogenerated line by `ember init`

Reply:
I know... we should change that... 😉

Reply:
haha, sounds good (when using `yarn` at least)